.gitignore

```diff
@@ -0,0 +1 @@
+./node_modules
```

Comment:
Hello Samuel, why did you add this to the gitignore file? We currently don't have node modules folder